demo-ui/public/stylesheets/ons.css

```diff
@@ -655,11 +699,42 @@ div.taColumn4H{
     background: #FF0000;
 }
 
+nav select {
+    display: none;
+}
+
+select {
+    font-family: "nta", Arial, sans-serif;
+    font-size: 1.1em
+}
+
 @media only screen and (max-width: 800px) {
     td {
         font-size: 0.6em;
         padding: 1px;
     }
+
+    nav ul     {
+        display: none;
+    }
+
+    nav select {
+        display: inline-block;
+        padding-bottom:2px;
+        padding-top:2px;
+        margin-bottom:4px;
+        margin-top:4px;
+        margin-left:4px;
+    }
+
+    details{
+        min-width: 400px;
+    }
+
+    details details div{
+        text-indent: 20px;
+        font-size:0.6em
```

Comment:
The font size change corrects the wrapping problem but makes it very hard to read in a browser.
In danger of sacrificing demoUI usability to enable small screen compatibility. I think you need to run these changes past Taiwo since the UI was previously agreed.